### PR TITLE
Bug in fix alpha single ended

### PR DESCRIPTION
**Pull request by bdestombe**
Single-ended measurements with `fix_alpha` failed due to a bug introduced in v2.0.0

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,16 @@
 Changelog
 =========
 
+dev
+---
+New features
+
+* Added function that aids in merging two single-ended measurements into a double-ended measurement. The added function checks solves for when measurements are missing in one channel but are present in the other.
```
Comment (BSchilperoort) [inline CHANGELOG.rst:9]:
```suggestion
* Improved the functionality of `merge_double_ended`, by adding a check that handles measurements missing in one channel while present in the other ([#171](https://github.com/dtscalibration/python-dts-calibration/pull/171))
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
+
+Bug fixes
+
+* Single-ended measurements with `fix_alpha` failed due to a bug introduced in v2.0.0
```
Comment (BSchilperoort) [inline CHANGELOG.rst:13]:
```suggestion
* Single-ended measurements with `fix_alpha` failed due to a bug introduced in v2.0.0 ([#173](https://github.com/dtscalibration/python-dts-calibration/pull/173)).
```

**Review by BSchilperoort — APPROVED** [review]
Thanks for fixing this! Was a test missing which prevented this bug from being caught? I would think a test that attempts to fix alpha for single ended calibration should have been present.

**Comment by bdestombe** [comment]
> Thanks for fixing this! Was a test missing which prevented this bug from being caught? I would think a test that attempts to fix alpha for single ended calibration should have been present.

You were right! The test with fix_dalpha was included, but fix_alpha was not. Just added the test.

**Pull request merged**